### PR TITLE
Fix number increase and validate button position on stock page

### LINKED ISSUE #21932: [UI-KIT] - Improve input in stock page
#### Describe the bug

Improve input in stock page

#### Steps to Reproduce

1. Go to admin -> Stock
2. Click on increase when the value is 0 
3. See error

**Screenshots**

https://drive.google.com/file/d/1XVM5nJqeMsoBkc5uk2etsaCSYydSblTG/view

#### Additional information

* PrestaShop version: develop
* PHP version: 7.1

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Stock page number button behavior was wrong
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #21932.
| How to test?  | See issue, also watch the global behavior of the page, in case it does break something (I checked but multiple checks are better than only one!)

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/21939)

### CODE REVIEW

**Review by Progi1984 — DISMISSED** [review]

**Review by Arman-Hosseini — DISMISSED** [review]

**Review by atomiix — DISMISSED** [review]

File: admin-dev/themes/new-theme/js/app/widgets/ps-number.vue [hunk 1]
```diff
@@ -26,12 +26,12 @@
...
   <div
     class="ps-number"
-    :class="{'hover-buttons': hoverButtons}"
+    :class="{ 'hover-buttons': hoverButtons }"
   >
     <input
       type="number"
       class="form-control"
-      :class="{'danger': danger}"
+      :class="{ danger: danger }"
```
Comment (PierreRambaud) [inline admin-dev/themes/new-theme/js/app/widgets/ps-number.vue]:
```suggestion
      :class="{ danger }"
```

**Review by sowbiba — DISMISSED** [review]

File: admin-dev/themes/new-theme/js/app/widgets/ps-number.vue (hunk 1 above)
Comment (NeOMakinG) [reply admin-dev/themes/new-theme/js/app/widgets/ps-number.vue]: Not related, but accepting it

**Review by PierreRambaud — APPROVED** [review]

**Review by atomiix — APPROVED** [review]

**Comment by khouloudbelguith** [comment]
Hi @NeOMakinG,

It is ok :heavy_check_mark: 
https://drive.google.com/file/d/1oU-tLgpWnWjulTQJn2QunaeDtmzuMJwn/view?usp=sharing

Thanks!

**Comment by Progi1984** [comment]
Thanks @NeOMakinG & @khouloudbelguith

**Pull request merged**